inc/telemetry/namespace.php

```diff
@@ -299,12 +299,23 @@ function get_environment_details() : array {
  * @return void
  */
 function load_segment_js() {
+	$page_name = get_admin_page_title();
+	switch ( $page_name ) {
+		case 'Home':
+			$page_name = 'Dashboard';
```

Comment:
Are the admin page title names translatable? This might not match if they are.

Additionally does 'Dashboard' refer to our replacement dashboard / accelerate? I would avoid having this kind of logic here ideally and aim to modify the admin page title in the context of the features that change it. I'm presuming `get_admin_page_title()` can be filtered?

Rather than looking at the page title string to decide whether to modify it we should look at the conditions for whether it should be modified, for instance on the docs pages `$_GET['page']` will equal `altis-documentation`: https://github.com/humanmade/altis-documentation/blob/master/inc/ui/namespace.php#L14-L46

Reply:
It looks to be that both the dashboard and accelerate when viewing the `/wp-admin` page returns as "Home".

`get_admin_page_title()` looks like it can be filtered, https://developer.wordpress.org/reference/hooks/admin_title/, should we wish for the "Home" value to be changed still.

Reply:
Yeah that's not surprising. So in the analytics plugin we could modify the page title using that filter if we're showing the accelerate dashboard.